packages/@n8n/config/src/configs/event-bus.ts

```diff
@@ -0,0 +1,31 @@
+import { Config, Env, Nested } from '../decorators';
+
+@Config
+class LogWriterConfig {
+	/** Number of event log files to keep */
+	@Env('N8N_EVENTBUS_LOGWRITER_KEEPLOGCOUNT')
+	readonly keepLogCount: number = 3;
+
+	/** Max size (in KB) of an event log file before a new one is started */
+	@Env('N8N_EVENTBUS_LOGWRITER_MAXFILESIZEINKB')
+	readonly maxFileSizeInKB: number = 10240; // 10 MB
+
+	/** Basename of event log file */
+	@Env('N8N_EVENTBUS_LOGWRITER_LOGBASENAME')
+	readonly logBaseName: string = 'n8nEventLog';
+}
+
+@Config
+export class EventBusConfig {
+	/** How often (in ms) to check for unsent event messages. Can in rare cases cause a message to be sent twice. `0` to disable */
+	@Env('N8N_EVENTBUS_CHECKUNSENTINTERVAL')
```

Comment:
should we rename this to `N8N_EVENTBUS_CHECK_UNSENT_INTERVAL` in v2?

Reply:
Let's please revisit and challenge as many env vars as reasonable in v2. Beyond naming, my impression is many of them are unused and should be removed. This one self-reports as buggy, which likely discourages even advanced users from using it.